### ISSUE TITLE
[Feature Request] Allow for more customization of the Media Library

### ISSUE THREAD

**Issue by Rafaeltheraven** [arybczak/ncmpcpp#339]
Right now, you can set the Primary Tag of the Media library to be anything you want, but the second and third rows will always have to be Album and Songs respectively. Now personally I prefer, for example, a workflow going genre -> AlbumArtist -> Album (songs be damned) and I expect there are others with similar specific needs. It would be nice if we could set these columns freely, instead of being restrained to a Tag -> Album -> Songs workflow.

**Comment by bostrr** [comment]
I agree! Have looked quite a few times how to sort the Album column by year while still displaying the names. Now it's just messy.

**Comment by benbuch** [comment]
agree. maybe a genre view where you can expand into sub genres.

https://raw.githubusercontent.com/beetbox/beets/master/beetsplug/lastgenre/genres-tree.yaml

**Comment by gokberkgunes** [comment]
Is there any update on this issue?

Does anybody have a patch/hack to implement something close to this behavior?
